om-2.0.rdf

```diff
@@ -8855,6 +8856,15 @@
     <om:hasDimension rdf:resource="&om;dimensionOne"/>
   </om:Unit>
 
+  <om:Unit rdf:about="&om;weightPercent">
+    <rdfs:label xml:lang="en">weight percent</rdfs:label>
+    <rdfs:label xml:lang="nl">gewicht procent</rdfs:label>
+    <rdfs:comment xml:lang="en">A mass-based percentage unit.</rdfs:comment>
+    <om:symbol>wt.%</om:symbol>
+    <rdf:type rdf:resource="&om;SingularUnit"/>
+    <om:hasDimension rdf:resource="&om;dimensionOne"/>
+  </om:Unit>

```

Comment:
Given that this is already a _mass_ fraction, I believe the `percent` unit is entirely sufficient, and `weightPercent` is redundant. In addition, the use of "weight" (as opposed to mass) is questionable (or at least outdated).

Reply:
@sm453 Hmmm but the unit that is provided to us is wt .%. I am not sure whether the conversion of wt .% to % will result in the loss of any information to the user.

Reply:
If you represent this as mass fraction with unit percent, then there is certainly no loss of information in the data stored. How this information is _displayed_ (e.g. on a web page) is a different story.

Reply:
Hmm okay. I will find a workaround for how it is displayed because the information will be displayed on a UI. Should be simple enough. I have removed weight percent.